hw/top_earlgrey/formal/conn_csvs/lc_escalate_en.csv

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run these checks with:
+#  ./util/dvsim/dvsim.py hw/top_earlgrey/formal/chip_conn_cfg.hjson
+
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+# Verify that lc_ctrl's lc_escalate_en_o signal is correctly connected to IPs.
+CONNECTION,LC_ESCALATE_EN_OTP,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_otp_ctrl,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_AON_TIMER,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_aon_timer_aon,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_SRAM_RET,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_sram_ctrl_ret_aon,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_FLASH,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_flash_ctrl,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_AES,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_aes,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_KMAC,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_kmac,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_OTBN,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_otbn,lc_escalate_en_i
+CONNECTION,LC_ESCALATE_EN_SRAM_MAIN,top_earlgrey.u_lc_ctrl,lc_escalate_en_o,top_earlgrey.u_sram_ctrl_main,lc_escalate_en_i
```

Comment:
nit: it might make sense to align the commas here so that we can tell the ports apart more easily:
```
CONNECTION, LC_ESCALATE_EN_OTP,       top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_otp_ctrl,          lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_AON_TIMER, top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_aon_timer_aon,     lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_SRAM_RET,  top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_sram_ctrl_ret_aon, lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_FLASH,     top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_flash_ctrl,        lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_AES,       top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_aes,               lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_KMAC,      top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_kmac,              lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_OTBN,      top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_otbn,              lc_escalate_en_i
CONNECTION, LC_ESCALATE_EN_SRAM_MAIN, top_earlgrey.u_lc_ctrl, lc_escalate_en_o, top_earlgrey.u_sram_ctrl_main,    lc_escalate_en_i
```

Reply:
Thanks Michael, I somehow thought CSV does not support spaces.... Not sure where that is coming from haha. But updated the file with extra spaces.

Reply:
Cool, thanks for updating!

The format is called "comma separated values" and not "space separated values" ;)